pkg/baseline/baseline.props

```diff
@@ -1,14 +1,18 @@
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="stable.packages.props" />
-  <Import Project="baseline.packages.props" />
-
   <PropertyGroup>
     <CoreFxBaseLinePackageName>Microsoft.Private.PackageBaseline</CoreFxBaseLinePackageName>
-    <CoreFxBaseLinePackageVersion>1.0.1-$(CoreFxExpectedPrerelease)</CoreFxBaseLinePackageVersion>
+    <!-- TODO: Move this back to 1.0.1 once https://github.com/dotnet/corefx/commit/63a868365904c6fbae18a3800c0b908b0f0944af is in -->
```

Comment:
This has to do with the change @ericstj just made to fix dotnet/buildtools#1013 which he has done with dotnet/buildtools#1020.
Once this PR is in I will be making a couple small changes to get Eric's change working for WCF.